packages/openapi/src/openapi-request-builder.spec.ts

```diff
@@ -76,6 +76,27 @@ describe('openapi-request-builder', () => {
     expect(response.data).toBe(dummyResponse);
   });
 
+  it('executeRaw executes a request without parameters and basePath explicitly set', async () => {
```

Comment:
[req] Please follow the convention here, it should be a complete sentence beginning with `it`

Reply:
Seems like other tests in the file also have this problem, will fix.